### ISSUE TITLE
Generators break between fusion-cli 1.9.0 and 1.11.4

### ISSUE THREAD

**Issue by vicapow** [fusionjs/fusion-cli#556]
to reproduce, add a generator somewhere in the current create-* scaffold and be sure to use it so it's not tree shaken.

    yarn create fusion-app fusion-debug-app-2
    cd fusion-debug-app-2

in `./src/pages/home.js`

````js
function* foo() {
  yield 1;
}

const fooSequence = foo();

//...

<FusionStyle>My app {fooSequence.next().value}</FusionStyle>
````

see that it compiles correctly using `yarn build-production`

update fusion deps: `yarn add fusion-cli@latest fusion-core@latest fusion-tokens@latest`

try to build-production again. `yarn build-production`

notice the following error:

````sh
error: (client) client-main-ae625f7157fa3294ebf9.js from UglifyJs
Unexpected token: operator (*) [./src/pages/home.js:49,0][client-main-ae625f7157fa3294ebf9.js:76,8]
````

**Comment by vicapow** [comment]
If I comment out these changes to exclude generators, the project compiles: https://github.com/fusionjs/fusion-cli/blame/bd180b7ec79ed45cd475f4159d28421d668a2b01/build/get-babel-config.js#L98

the project compiles but doesn't run with the error:

````js
home.js:49 Uncaught ReferenceError: regeneratorRuntime is not defined
    at Module../src/pages/home.js (home.js:49)
    at __webpack_require__ (bootstrap:782)
    at fn (bootstrap:150)
    at Module../src/root.js (pageNotFound.js:11)
    at __webpack_require__ (bootstrap:782)
    at fn (bootstrap:150)
    at Module../src/main.js (main.js:2)
    at __webpack_require__ (bootstrap:782)
    at fn (bootstrap:150)
    at reload (client-entry.js:16)
```

**Comment by rtsao** [comment]
Thanks for the reproductions steps. This is fixed by https://github.com/fusionjs/fusion-cli/pull/557 and released in `fusion-cli@1.11.5`
